src/test/scala/nl.knaw.dans.easy.deposit/docs/AgreementsSpec.scala

```diff
@@ -43,29 +46,38 @@ class AgreementsSpec extends TestSupportFixture {
         acceptDepositAgreement = true,
         privacySensitiveDataPresent = PrivacySensitiveDataPresent.unspecified
       )
-    ) should matchPattern { case Failure(InvalidDocumentException(_, e)) if e.getMessage == "Please set PrivacySensitiveDataPresent" => }
+    ) should matchPattern {
+      case Failure(InvalidDocumentException(_, e)) if e.getMessage == "Please set PrivacySensitiveDataPresent" =>
+    }
   }
+
+  private lazy val triedSchema: Try[Schema] = AgreementsXml.loadSchema
+
   "schema validation" should "succeed" in {
-    assume(AgreementsXml.triedSchema.isAvailable)
+    assume(triedSchema.isAvailable)
     AgreementsXml(
       "user",
       DateTime.now,
       DatasetMetadata().copy(
         acceptDepositAgreement = true,
         privacySensitiveDataPresent = PrivacySensitiveDataPresent.no
       )
-    ).flatMap(AgreementsXml.validate) should matchPattern { case Success(_) => }
+    ).flatMap(triedSchema.validate) should matchPattern {
```

Comment:
why not use `shouldBe a[Success[_]]` if you're not going to check anything else?